chapters/remote/deploy-jinad-on-aws-with-terraform.md

```diff
@@ -0,0 +1,125 @@
+# Deploying JinaD on AWS With Terraform
+
+## Overview
+
+Here is a [terraform module jinad-aws](https://registry.terraform.io/modules/jina-ai/jinad-aws/jina/latest) to deploy `JinaD` on AWS by terraform.
+
+With this terraform module you can:
+- Custom the type and size of the root disk
+- Deploy multiple AWS EC2 instance
+- Execute extra command line
```

Comment:
Remove this point

Reply:
uh-ha, which point?

Reply:
Ha good question, 

`Execute extra command line`